### DISCUSSION TITLE
Masonry Layout of devices in big screen.

### DISCUSSION THREAD

**Discussion by ronaldohoch** [responsively-org/responsively-app#1206] [Q&A] [answered]
Hi guys.

First of all, thank you for this project, it saves a lot of time :D

Back in 2010's we had the masonry plugin to create automatic mosaic layouts before display: grid..
![image](https://github.com/responsively-org/responsively-app/assets/2287371/dc543983-da34-48b8-8a05-4d811ee953ad)

In a vertical screen, only shows 3 devices and i keep with a big space below..
Also, in a normal big screen, we have space below.
I didn't find any way in the app to configure it. 

Is it possible?

![image](https://github.com/responsively-org/responsively-app/assets/2287371/ed6ffb72-4aca-4ddf-ba79-a0939521d4ea)

![image](https://github.com/responsively-org/responsively-app/assets/2287371/15aac70f-9b8d-4889-980d-e827fbf62f1d)

also, my screen sizes are:
2560x1440 and 1440x2560

**Comment by manojVivek** [comment, marked as answer]
Hi @ronaldohoch, We have thought about this but haven't got to implement it, I have created an issue where we can track the progress on this. 
Issue: https://github.com/responsively-org/responsively-app/issues/1207

**Reply by ronaldohoch** [reply]
Thanks for the reply, I'll be following the issue

**Comment by violetadev** [comment]
Solved here: https://github.com/responsively-org/responsively-app/pull/1311
